packages/web/src/common/store/notifications/types.ts

```diff
@@ -174,35 +174,54 @@ export type TierChange = BaseNotification & {
   user: User
 }
 
-export type TipReaction = BaseNotification & {
-  type: NotificationType.TipReaction
+// TODO: when we support multiple reaction types, reactedToEntity type
```

Comment:
nit: We could make a `Entity.Tip` type so we're ready?

Reply:
let's cross the bridge when we come to it